work/worker.go

```diff
@@ -932,9 +935,14 @@ func (env *Task) commitBundleTransaction(bundle *builder.Bundle, bc BlockChain,
 			}
 			*env.state = *lastSnapshot
 			env.header.GasUsed = gasUsedSnapshot
+			env.tcount = tcountSnapshot
+			if err == nil {
```

Comment:
How about ignoring the original `err` and return `kerrors.ErrRevertedBundleByVmErr` always?

Reply:
I think we don't have to override this for proper logging and error counting.
Also, in the case of NonceTooLow, I think it is enough to use shift instead of pop, even if it is a tx in a bundle.
https://github.com/kaiachain/kaia/pull/282/files#diff-aa53dfe64c81ad7f4673af6857ca8ae7873bb18224202adb09f765ce4f458411R812-R816

Reply:
Ok, thanks.